etip/trackers/management/commands/compare_with_exodus.py

```diff
@@ -0,0 +1,99 @@
+from django.core.management.base import BaseCommand, CommandError
+import requests
+
+from trackers.models import Tracker
+
+EXODUS_API_BASE_URL = 'https://reports.exodus-privacy.eu.org'
+EXODUS_API_BASE_PATH = '/api/trackers'
+FIELDS_TO_LOOKUP = [
+    'name',
+    'code_signature',
+    'network_signature',
+    'website'
+]
+LOOKUP_IS_EXACT = 'EXACT_MATCH'
+LOOKUP_IS_DIFFERENT = 'DIFFERENT'
+LOOKUP_MULTIPLE = 'MULTIPLE_IN_ETIP'
+LOOKUP_NOT_FOUND = 'NOT_FOUND_IN_ETIP'
+
+
+class Command(BaseCommand):
+    help = 'Compare trackers stored in Exodus with trackers from ETIP DB'
+
+    def handle(self, *args, **options):
+        exodus_trackers = self.get_all_from_exodus()
+        self.stdout.write(
+            'Retrieved {} trackers from Exodus'.format(len(exodus_trackers))
+        )
+        self.count_from_etip()
+        self.lookup_trackers(exodus_trackers)
+
+    def __exact_lookup(self, tracker_details):
```

Comment:
I think maybe adding docstrings for this function (maybe for the others too) to make it clearer what it does and returns.

Reply:
The `__exact_lookup` method was badly named, had too many responsabilities and was overly complicated =) Refactored it into `find_etip_tracker`, should be clearer. Please tell me if you think it still needs docstrings!

["Don't document bad code; rewrite it"](https://twitter.com/voidspace/status/252783747785687040) :p